pkg/client/simple/vfsclientset/import_known_versions.go

```diff
@@ -20,13 +20,24 @@ package vfsclientset
 import (
 	"fmt"
 
-	_ "k8s.io/kops/pkg/apis/kops/install"
-	_ "k8s.io/kubernetes/pkg/api/install"
-	"k8s.io/kubernetes/pkg/apimachinery/registered"
+	"k8s.io/client-go/pkg/api"
+	_ "k8s.io/client-go/pkg/api/install"
+	_ "k8s.io/client-go/pkg/apis/apps/install"
+	_ "k8s.io/client-go/pkg/apis/authentication/install"
+	_ "k8s.io/client-go/pkg/apis/authorization/install"
+	_ "k8s.io/client-go/pkg/apis/autoscaling/install"
+	_ "k8s.io/client-go/pkg/apis/batch/install"
+	_ "k8s.io/client-go/pkg/apis/certificates/install"
+	_ "k8s.io/client-go/pkg/apis/extensions/install"
+	_ "k8s.io/client-go/pkg/apis/policy/install"
+	_ "k8s.io/client-go/pkg/apis/rbac/install"
+	_ "k8s.io/client-go/pkg/apis/settings/install"
+	_ "k8s.io/client-go/pkg/apis/storage/install"
```

Comment:
I guess they split this up so we don't need to import everything.

But ... also the kops import is missing now?